agents/lakeshore372/LS372_agent.py

```diff
@@ -104,20 +176,35 @@ def init_lakeshore_task(self, session, params=None):
 
     def start_acq(self, session, params=None):
 
-        with self.lock.acquire_timeout(0, job='acq') as acquired:
-            if not acquired:
-                self.log.warn(f"Could not start init because "
-                              f"{self.lock.job} is already running")
+        with self._acq_proc_lock.acquire_timeout(timeout=0, job='acq') \
+             as acq_acquired, \
+             self._lock.acquire_timeout(job='acq') as acquired:
+            if not acq_acquired:
+                self.log.warn(f"Could not start Process because "
+                              f"{self._acq_proc_lock.job} is already running")
+                return False, "Could not acquire lock"
+            if not acq_acquired:
```

Comment:
should check `acquired` instead of `acq_acquired`

Reply:
good catch... fixed.